packages/strapi-design-system/src/index.ts

```diff
@@ -44,7 +44,7 @@ export * from './Flex';
 export * from './Searchbar';
 export * from './Select';
 export * from './SimpleMenu';
-export * from './Stack';
+export * from './Stack/index.ts';
```

Comment:
🤔 Do you really need to precise `index.ts` ?

Reply:
I think my IDE did that, thanks for catching.